files/usr/lib/jeos-firstboot

```diff
@@ -361,9 +361,9 @@ for d in ${candidates[@]} ; do
         break # one is enough
 done
 
+trap 'call_module_hook cleanup' EXIT
 call_module_hook systemd_firstboot
 call_module_hook post
```

Comment:
This needs to be moved to the place where the code previously was.

Reply:
Actually not if we want to make it run always on exit even if something very bad happens. As script is only run only at the end.

Reply:
I'm referring to `call_module_hook post`, not the cleanup trap.

Reply:
Did I messed something up?

Reply:
Now it should be in correct place?

Reply:
Yup!